modules/web/src/app/shared/components/labels/component.ts

```diff
@@ -67,6 +69,20 @@ export class LabelsComponent implements OnInit, OnChanges {
     return this.oneLineLimit && this.hideExtraLabels;
   }
 
+  getLabelKey(label: string): string {
+    if (label.length > this._maxLabelLength) {
```

Comment:
I don't have any better solution ATM but i am thinking in case label/value with 21 char ..... so we will be shwoing 20 char... and then but tooltip shows full value which make sense.

TBH, If we should handle with CSS or making more dynamic as "@Input"  but for now it works fine

  private readonly _maxLabelLength = 20;
  private readonly _displayedLength = 17;


Approach-2

OR  can we Add Input `@Input() disableTruncation` and it should work for specific componenent where needed otherwise default functionality should work?


thoughts?